nncf/experimental/onnx/graph/metatypes/onnx_metatypes.py

```diff
@@ -29,25 +28,51 @@ def get_all_aliases(cls) -> List[str]:
         return cls.op_names
 
 
+class OpWeightDef:
+    """
+    Contains information about the weight of operation.
+    """
+
+    def __init__(self, weight_port_id: Optional[int], channel_axis: int, bias_port_id: Optional[int]):
```

Comment:
Some arguments have `Optional`, but don't have a default parameter value.  Also the docstring does not say what happens when each optional parameter is not given.

Reply:
Done